OrbitCore/Path.cpp

```diff
@@ -227,14 +227,25 @@ std::string Path::GetDirectory(const std::string& a_FullName) {
 }
 
 std::string Path::GetParentDirectory(std::string a_Directory) {
-  if (a_Directory.size() < 1) return "";
+  if (a_Directory.empty()) return "";
   std::replace(a_Directory.begin(), a_Directory.end(), '\\', '/');
   wchar_t lastChar = a_Directory.c_str()[a_Directory.size() - 1];
   if (lastChar == '/') a_Directory.erase(a_Directory.size() - 1);
 
   return GetDirectory(a_Directory);
 }
 
+std::string Path::JoinPathParts(const std::vector<std::string>& parts) {
```

Comment:
Suggestions for a nicer name are welcome.

Reply:
JoinPath?

Reply:
Yeah, why not